geniza/corpus/tests/test_corpus_models.py

```diff
@@ -1644,6 +1645,65 @@ def test_document_merge_with_log_entries(document, join):
     assert " [PGPID %s]" % join_pk in moved_log.change_message
 
 
+def test_document_merge_with_dates(document, join):
+    editor = User.objects.get_or_create(username="editor")[0]
+
+    # clone join twice for additional merges
+    join_clone = Document.objects.get(pk=join.pk)
+    join_clone.pk = None
+    join_clone.save()
+    join_clone_2 = Document.objects.get(pk=join.pk)
+    join_clone_2.pk = None
+    join_clone_2.save()
+
+    # create some datings; doesn't matter that they are identical, as cleaning
+    # up post-merge dupes is a data cleanup task. unit test will make sure that
```

Comment:
Does this mean manual data cleanup?  Shouldn't be that hard to detect exact duplicates by implementing an `__eq__` method on the `Dating` object like we did on `Foonotes` (IIRC), but you all would have to determine how likely or valuable that is.

Reply:
Team agree this is fine as a manual data cleanup task, but can revisit if it starts to become a problem—thanks for the solution!